bin/gce-delete-instances

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
``gce-delete-instace --all`` is good to me. Actually this is the first idea came to my mind. I did not propose this version because this needs to largely refactor the code. Besides, two-script implementation is more readable (to me) but may cause confusion to users as what you mentioned. I will create another PR soon for the ``gce-delete-instace --all`` and let's compare which one is better.